src/sources/user_agent.test.ts

```diff
@@ -1,15 +1,12 @@
-import { getBrowserVersion } from '../../tests/utils'
+import { getBrowserEngineVersion } from '../../tests/utils'
 import getUserAgent from './user_agent'
 
 describe('Sources', () => {
   describe('userAgent', () => {
     it('returns an expected value', () => {
       const value = getUserAgent()
-      const version = getBrowserVersion() ?? { major: 0, minor: 0 }
-
-      // We do not want to call ua-parser-js in these tests, as those likely
-      // use navigator.userAgent internally.
```

Comment:
I've removed the comment because `getBrowserVersion` and `getBrowserEngineVersion` use ua-parser-js under the hood